src/main/java/com/github/luben/zstd/ZstdInputStream.java

```diff
@@ -46,10 +46,24 @@
 
     // The main constructor / legacy version dispatcher
     public ZstdInputStream(InputStream inStream) {
+        this(inStream, BufferPool.get(srcBuffSize), null);
+    }
+
+    /**
+     * construct a ZstdInputStream with custom byte array. This construction is useful when you prefer to be able to
+     * manage the buffers. You can call {@link ZstdInputStream#srcBuffSize} to get the recommended size.
+     * @param inStream input stream
+     * @param src byte array
+     */
+    public ZstdInputStream(InputStream inStream, byte[] src) {
```

Comment:
Can we rename "src" -> "buffer" or something similar?

Reply:
copy that.